CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
-# dbt_fivetran_log v1.0.1
+# dbt_fivetran_log v1.1.0
+
+PR # includes the following updates:
+## 🚨 Feature Updates (Breaking Change) 🚨 
+The below change was made to an incremental model. As such, a `dbt run --full-refresh` will be required following an upgrade to capture the new column.
+-  Added `schema_name` to the `fivetran_platform__audit_table` end model. This schema name field is captured from the `message_data` JSON within the `log` source table. In cases where the schema_name is not provided a coalesce was added to replicate the `connector_name` as the `schema_name.
```

Comment:
should we note that the row count may change but that it is more correct/granular?